pages/samvera/developer_resources/patterns/presenters.md

```diff
@@ -0,0 +1,223 @@
+---
+title: "Presenters Design Pattern"
+permalink: patterns-presenters.html
+folder: samvera/developer_resources/patterns/presenters.md
+sidebar: home_sidebar
+a-z: ['Design Pattern - Presenters', 'Presenters Design Pattern']
+keywords: Design Patterns
+tags: [development_resources]
+categories: Design Patterns
+---
+
+# Understanding the Presenters Design Pattern
+
+<ul class='info'><li><b>Primary Goal: Isolate views from business logic code.</b></li></ul>
+
+## Characteristics:
+* Presenters are used to hide implementation details from views.  They serve as a go-between for controllers and views.
+* Presenters provide access to controller instance variables.
+* `@presenter` should be the only instance variable accessed in the view.
+* `@presenter` can be passed as `presenter` to partials rendered from the primary view.
+* The same presenter can be shared by multiple views and partials. 
+* Presenters can be used with any kind of view.  Although, they are not used with new/edit forms.  The Forms pattern performs a similar role for the forms.
```

Comment:
Grammar is awkward here. The second partial sentence should be combined with the first.